Cython/Compiler/Nodes.py

```diff
@@ -362,6 +364,32 @@ def annotate(self, code):
         code.globalstate.directives = old
 
 
+class CPPNode(Node):
```

Comment:
`CPP` generally suggests C++, not C PreProcessor.

Reply:
Right, I'll rename that but the node should eventually go away.